src/tests/ftest/osa/online_extend.py

```diff
@@ -142,7 +142,8 @@ def run_online_extend_test(self, num_pool, racer=False,
             display_string = "Pool{} space at the End".format(val)
             self.pool = pool[val]
             self.pool.display_pool_daos_space(display_string)
-            self.run_ior_thread("Read", oclass, test_seq)
+            if app_name == "ior":
+                self.run_ior_thread("Read", oclass, test_seq)
```

Comment:
I'm not familiar with this test, but do we need something like this?
```
else:
    self.run_mdtest_thread()
```

Reply:
oh, I think that is for verification as I thought. I am not sure if there are separate verification for mdtest? @rpadma2

Reply:
Presently, mdtest doesn't do separate write/read. It just runs mdtest on the same container read/write together. That's all... In future, we can enhance it (if needed). IOR is only used as separate write and read now.

Reply:
We should run mdtest with read & stat to verify the tree. Let's create another ticket for this.